src/Http/Http/src/Features/RequestBodyPipeFeature.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO.Pipelines;
+
+namespace Microsoft.AspNetCore.Http.Features
+{
+    public class RequestBodyPipeFeature : IRequestBodyPipeFeature
```

Comment:
Move these to the Http assembly.

Reply:
They are in the Http Folder/Assembly. Different namespace though.

Reply:
Ah missed that